server/src/databases/handlers/surveyDBHandler.test.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getServerConfig } from '../../utils/getConfig';
+import SurveyDBHandler from './surveyDBHandler';
+
+describe('Test surveyDBHandler', () => {
+  beforeEach(() => {
+    process.env.VV_COSMOS_DB_ENDPOINT = 'https://testinghost/';
+  });
+
+  afterEach(() => {
+    delete process.env.VV_COSMOS_DB_ENDPOINT;
+  });
+
+  test('Test init', async () => {
+    const config = getServerConfig();
```

Comment:
Similar comment here about calling external dependencies in unit tests. We should create mock configs instead where we control the actual values.

Reply:
Yes I will update the test.